portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,30 @@ function addRandomFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = randomFact;
 }
+
+function getFromServlet(){
+    console.log("Fetching from sevlet");
+
+    fetch('/data').then(handle);
+}
+
+function handle(response){
+    console.log("Handling promise");
+
+    response.text().then(addToSite);
+}
+
+function addToSite(saying){
+    console.log("Adding " + saying + " to site");
+
+    const sayingContainer = document.getElementById('saying-container');
+    
+    if(sayingContainer == null){
+       console.assert(document.getElementById('saying-container'), "You have no element with ID 'saying-container'"); 
+    }
```

Comment:
No need for an if statement.  console.assert only triggers if the first argument is falsey (i.e. null)